packages/web-api-scan-job-manager/package.json

```diff
@@ -3,7 +3,7 @@
     "version": "1.0.0",
     "description": "This project welcomes contributions and suggestions.  Most contributions require you to agree to a Contributor License Agreement (CLA) declaring that you have the right to, and actually do, grant us the rights to use your contribution. For details, visit https://cla.microsoft.com.",
     "scripts": {
-        "build": "webpack --config ./webpack.config.js --colors",
+        "build": "webpack --config ./webpack.config.js --config-name web-api-scan-job-manager --colors",
```

Comment:
this parameter is not used if the webpack config doesnt contain array of configuration

Reply:
we still add it in other packages

Reply:
yes. but it is wrong. I have fixed it in some places. but not everywhere. you don't have to carry over the issue since we know that it doesnt work.

Reply:
how doesn't it work? can you elaborate?